src/ansys/aedt/core/modeler/circuits/primitives_circuit.py

```diff
@@ -780,6 +780,8 @@ def create_component(
             The default is ``False``.
         global_netlist_list : list, optional
             The default is ``None``, in which case an empty list is passed.
+        page: int, optional
+            Schematics page number. The default value is ``1``.
```

Comment:
```suggestion
            Schematic page number. The default value is ``1``.
```
Is there a need for an extra 's' ?